lib/sitemap.d.ts

```diff
@@ -74,13 +68,16 @@ export declare class Sitemap {
      *  Create sitemap xml
      *  @param {Function}     callback  Callback function with one argument — xml
      */
-    toXML(callback: any): string;
+    toXML(callback: Callback<Error, string>): void
+    toXML(): string;
     /**
      *  Synchronous alias for toXML()
      *  @return {String}
      */
     toString(): string;
-    toGzip(callback?: Function): any;
+  // returns Buffer | void - not sure how to import
+    // https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/v10/globals.d.ts#L229
+    toGzip(callback?: (error: Error | null, result: Buffer) => void): any;
```

Comment:
You can add 2 definitions of the function (same as toXML): 

```
toGzip(): Buffer;
toGzip(callback: Callback<Error, Buffer>): void;
```

Since `Buffer` does not exist in browser, you have to add `@types/node` module to `devDependencies`.